scopus-import/src/main/java/no/sikt/nva/scopus/conversion/CristinContributorExtractor.java

```diff
@@ -59,26 +63,26 @@ private static Identity generateContributorIdentityFromCristinPerson(Person cris
     }
 
     private static ContributorVerificationStatus generateVerificationStatus(Person cristinPerson) {
-        return cristinPerson.getVerified() ? ContributorVerificationStatus.VERIFIED
+        return Boolean.TRUE.equals(cristinPerson.getVerified())
+                   ? ContributorVerificationStatus.VERIFIED
                    : ContributorVerificationStatus.NOT_VERIFIED;
     }
 
-    private static List<Organization> generateOrganizations(Set<Affiliation> affiliations,
-                                                            no.sikt.nva.scopus.conversion.model.cristin.Organization organization) {
+    private static List<Organization> generateOrganizations(
+        Set<Affiliation> affiliations,
+        no.sikt.nva.scopus.conversion.model.cristin.Organization organization) {
 
         var organizations = createOrganizationsFromCristinPersonAffiliations(affiliations);
-        if (nonNull(organization)) {
-            organizations.add(createOrganizationFromAuthorGroupTpAffiliation(organization));
-            return organizations;
-        }
-        return organizations;
+        var organisationFromAuthorGroupTp = createOrganizationFromAuthorGroupTpAffiliation(organization);
+        return Stream.concat(organizations, organisationFromAuthorGroupTp)
+                   .filter(Objects::nonNull)
+                   .toList();
     }
 
     @NotNull
-    private static List<Organization> createOrganizationsFromCristinPersonAffiliations(Set<Affiliation> affiliations) {
-        return affiliations.stream()
-                   .map(CristinContributorExtractor::convertToOrganization)
-                   .collect(Collectors.toList());
+    private static Stream<Organization> createOrganizationsFromCristinPersonAffiliations(
+        Set<Affiliation> affiliations) {
+        return affiliations.stream().map(CristinContributorExtractor::convertToOrganization);
```

Comment:
Sending streams around

Reply:
As long as the same stream is not consumsed more than once.